src/notyf.scss

```diff
@@ -125,6 +125,52 @@ $toast-padding: 15px;
         left: 10px;
       }
     }
+
```

Comment:
This looks almost exactly like the block for `&--success`, right? can you please DRY it a bit? like
```scss
&--success, &--info, &--warning {
  ...
}
```

Better yet, we probably don't need those 3 classes if they do the same. Maybe there can be a shared class `&--base` with this. What do you think?

Reply:
Agreed, base makes sense. Let's do that!